### ISSUE TITLE
Replace various plain text fields with the HTML field

### ISSUE THREAD

**Issue by pradumangoyal** [IMGIITRoorkee/omniport-frontend-groups#2]
The groups service is like a display profile page for the groups. Some of the fields like Post text, description text are expected to have some external links, and some formatting in it. The motive of this PR is to replace those text area boxes with TinyMCE editor and required changes on the display side.

**Comment by Prernaswarna** [comment]
I would like to do it

**Comment by pradumangoyal** [comment]
Yes, please proceed. You can check out the implementation in the maintainer_site. @algomaster99

**Comment by pradumangoyal** [comment]
@Prernaswarna any updates?

**Comment by Prernaswarna** [comment]
Yeah i have made changes to post and i am also supposed to make changes to about ? Or mission ? Or both ?

**Comment by Prernaswarna** [comment]
@pradumangoyal  Sorry I forgot to mention you

**Comment by pradumangoyal** [comment]
Hey, yes please change the about and mission section as well.

**Comment by Prernaswarna** [comment]
@pradumangoyal Hey I have made pull requests here and in the backend
